syntax = "proto3";

import "onnx.proto";
import "solvers_dataset.proto";

message Empty {}

message File {
    string file = 1;
}

service Regresion {
    rpc StreamLogs ( Empty) returns (stream File){}
    rpc MakeRegresion (  dataset.DataSet) returns (tensor_onnx.ONNX) {}  
}